app/services/document_builder.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
```

Comment:
This is just moved from Indexer and renamed to DocumentBuilder. No new code was added.